templates/terraform/custom_expand/mlengine_job_id.erb

```diff
@@ -0,0 +1,25 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2018 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+#%>
+func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d TerraformResourceData, config *Config) (interface{}, error) {
```

Comment:
Why are we adding extra information into the job id instead of passing what the user gave us through?

Reply:
We are adding timestamp of the job. Similar to to gcloud command does as per recommendation. Similar to the link https://cloud.google.com/ml-engine/docs/scikit/training-scikit-learn

JOB_NAME=census_training_$(date +"%Y%m%d_%H%M%S")

Job name must be unique each time

Reply:
Will review with team and remove it.

Reply:
removed this